components/Onboarding.js

```diff
@@ -170,16 +148,10 @@ class Onboarding extends PureComponent {
   onHightlightButtonPress = () => {
     const { step, setStep } = this.props;
 
-    switch (step) {
-      case 8:
-        NavigationService.openDrawer();
-        break;
-      case 9:
-        NavigationService.navigate('Invites');
-        break;
-      default:
-        break;
+    if (STEPS[ORDERED_STEPS[step]].buttonLogic) {
+      STEPS[ORDERED_STEPS[step]].buttonLogic();
```

Comment:
Should these actions be thunks?

Reply:
@smaclell What benefit would that give?

Reply:
Consistency? More of a though experiment really. I was thinking you could even encapsulate the logic around these things. You could always dispatch and then internally trigger the `buttonLogic`.